src/main/java/it/pagopa/afm/marketplacebe/repository/BundleRepository.java

```diff
@@ -16,6 +16,7 @@ public interface BundleRepository extends CosmosRepository<Bundle, String> {
     Optional<Bundle> findById(String idBundle);
     Optional<Bundle> findById(String idBundle, PartitionKey idPsp);
 
+    Optional<Bundle> findByIdPspAndName(String idPsp, String name);
```

Comment:
it is better to improve performances the following query
`Optional<Bundle> findByNameAndIdPsp(String name, PartitionKey idPsp);`